app/helpers/document_url_helper.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-module EditionUrlHelper
+module DocumentUrlHelper
```

Comment:
Was there a reason not to go with just UrlHelper? It wouldn't surprise me too much if that caused problems.

I'm not too sure `DocumentUrlHelper` really explains this well, maybe it's ExternalUrlHelper since everything here is to URL's outside the app.

@benthorner any thoughts/ideas/preferences?

Reply:
Hmm...it looks like this change was made so we can use the new method in a view. Since the method is only one line, can we just duplicate it into the view?

Reply:
Can do but then we need to remember an interdependency between the host name used in the edition_preview_url method and in the view since they need to point to the same host.

Reply:
`UrlHelper` felt too generic. Not very opinionated about the naming as long as it makes sense to everyone.

@benthorner I think the idea was to avoid duplication, but if it doesn't fit within the helper I'm fine duplicating it into the view.

Reply:
Updated and renamed the helper to `ExternalUrlHelper`